addons/payment_mollie_official/models/provider_log.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+import datetime
+import logging
+from odoo import fields, models, api, _
+
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+_logger = logging.getLogger(__name__)
+
+
+class ProviderLog(models.Model):
```

Comment:
this model should be removed
the message on the tx  + testing in the front end when deploying should be enough to diagnose problems